main.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import re
+from lxml import html
+from lxml.cssselect import CSSSelector
+import csv
+
+urlinit = 'http://www.schooldays.ie'
+
+def regionurl():
+
+#Collect the content of each schooldays.ie urls for region Dublin-1, Dublin-2 ....Dublin-18 and calls schollurl() function,
+#Create the CSV file with the header
+
+    header = ['School Name', 'Address', 'Postal District', 'Year', 'Total Sitting Leaving', #csv reader
+          'Total Main Colleges']
+    with open('./schoolfile', 'a') as myfile: #write the csv reader
```

Comment:
I think you could have this value (`schoolfile`) since it seems a constant.. if in the future you want to change it.. you'll need only to change in one place. Other options is passing it a a parameter....

Reply:
ok